tests/test_calculation_accuracy_metric.py

```diff
@@ -1,11 +1,13 @@
+import numpy as np
+
 from flownet.ahm._ahm_iteration_analytics import accuracy_metric
 
 
 def test_calculation_accuracy_metric() -> None:
     assert (
         accuracy_metric(
-            [[0.5, 0.5, 0.5], [0.5, 0.5, 0.5]],
-            [[0.4, 0.4, 0.4], [0.6, 0.6, 0.6]],
+            np.array([[0.5, 0.5, 0.5], [0.5, 0.5, 0.5]]),
+            np.array([[0.4, 0.4, 0.4], [0.6, 0.6, 0.6]]),
             "RMSE",
         )
         == 0.09999999999999998
```

Comment:
Maybe this test should use something like [`math.isclose`](https://docs.python.org/dev/library/math.html#math.isclose) in order to not get a false negative due to floating point issues?

Reply:
good point! I have never used this, but will take a look into it